src/contexts/NotificantionManager/notification.util.ts

```diff
@@ -0,0 +1,43 @@
+import { ToastVariant } from '@chili-publish/grafx-shared-components';
+import { Notification } from './Notification.types';
+
+const notificationTypes = [
```

Comment:
why it's not part of the shared component?

Reply:
it can be moved indeed, but the priorities that are here belongs to platform and may change based on the project.